config/webpack/environment.js

```diff
@@ -1,3 +1,19 @@
 const { environment } = require('@rails/webpacker')
 
-module.exports = environment.toWebpackConfig()
+// Add global webpack configs here
+
+const babelLoader = environment.loaders.get('babel')
+babelLoader.test = /\.jsx?|.es6?|.spec.js?$/
```

Comment:
I'm pretty sure you can remove testing .es6

Reply:
And remember the packs are .js

Reply:
However this is exactly what's currently in `master`:
```
{
        test: /\.jsx?|.es6?|.spec.js?$/,
        exclude: /(node_modules)/,
        use: [
          {
            loader: 'babel-loader',
            options: {
              presets: ['es2015', 'react'],
              plugins: ['transform-object-rest-spread', 'transform-class-properties', 'transform-es2015-destructuring']
            }
          }
        ]
      }
```
So what should it be?

Reply:
But we are changing what's there, since we migrated all our .es6 assets, right? this is a good PR to adjust that too :)

Reply:
Got it!